### PR TITLE
Fix reported issues

### PR DESCRIPTION
**Pull request by pivagabriel**
## What does this Pull Request Do?

- Fix reported issues

## How is this tested?

Validate this issues:

https://github.com/PipeWebMonetization/analytics-tool/issues/10
https://github.com/PipeWebMonetization/analytics-tool/issues/11
https://github.com/PipeWebMonetization/analytics-tool/issues/12
https://github.com/PipeWebMonetization/analytics-tool/issues/13
https://github.com/PipeWebMonetization/analytics-tool/issues/14

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -39,6 +36,25 @@ const Dashboard: NextPage = () => {
   const [pluginIds, setPluginIds] = useState<PluginIdDocument[]>([]);
   const [selectedPluginId, setSelectedPluginId] = useState<string>("");
   const [selectedYear, setSelectedYear] = useState<string>("2022");
+  const [hasMonetizationListener, setMonetizationListener] = useState<boolean>(false);
+  const batcher = new Batcher();
+
+  useEffect(() => {
+    if (!hasMonetizationListener) {
+      setMonetizationListener(true);
```
Comment (EijiTomonari) [inline src/pages/index.tsx:44]: If you are already setting hasMonetizationListener to true whener the app renders, why do you need the state hasMonetizationListener?

Comment (pivagabriel) [reply src/pages/index.tsx:44]: Actually I set to false as default, and I did this validation to not overwrite the listener.

Comment (EijiTomonari) [reply src/pages/index.tsx:44]: Ok. You set the default to false. But the useEffect runs as soon as the page is loaded and set it to true. Why not set the default value to true?

Comment (EijiTomonari) [reply src/pages/index.tsx:44]: Move batcher to inside the useEffect

**Review by EijiTomonari — APPROVED** [review]

**Pull request merged**